### ISSUE TITLE
resume 

### ISSUE THREAD

**Issue by Prograam** [stoppieboy/stoppieboy.github.io#9]
google drive resume not accesseble to everythone ,

**Comment by stoppieboy** [comment]
> google drive resume not accesseble to everythone ,

yes i have done it intentionally because i wasn't sure if its safe to distribute my contact information publicly to everyone on the internet
